documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc

```diff
@@ -610,6 +610,12 @@ NOTE: This list is not exhaustive - there may be codes in use that are not in th
 | 512MB
 | Sony UK
 
+| b04170
```

Comment:
I think this should be b04171, it's a revision 1.1 board. But I don't have one to check.

Reply:
It's revision 1.0 - revision 1.1 boards might follow

Reply:
```suggestion
| b04171
```